javascript/react/src/components/ArqueroDetailsList/ArqueroDetailsList.tsx

```diff
@@ -75,6 +77,8 @@ export const ArqueroDetailsList: React.FC<ArqueroDetailsListProps> = memo(
 			defaultSortDirection,
 		)
 
+		const _fields = useFields(table, fields)
```

Comment:
why the underscore here? That usually indicates a parameter variable that's being ignored

Reply:
We've used it in a variety of places inside of functions to indicate internal, but I agree it now conflates with ignored